lib/other_netns.c

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
Again we use <system>_<subsystem>.c Shouldn't this be ns_other.c?

Reply:
netns_other.c ?

Reply:
fine with me.